composer.json

```diff
@@ -19,7 +19,7 @@
   },
   "require": {
     "php": ">=5.6",
-    "typo3/cms": ">=7.6,<9.0",
+    "typo3/cms-core": ">=7.6,<9.0",
```

Comment:
Thanks for your PR @koehnlein.

You're absolutely right!
I would like to change the constraint to `"typo3/cms-core": "^7.6 || ^8.7"` thus is fits the LTS.
May you add this or should I do directly and close your PR?